### DISCUSSION TITLE
Success page after form submission

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#212] [Ideas]
After successful form submission, there's nothing out of the box to show a final page which has some kind of success message. Given how common this is for forms I think it should be built-in, even if it's something basic only.
Alternatively, some example on how to do it manually.

**Comment by adamberecz** [comment]
Not really a success page, but here's how you can display a success message using built-in Vueform feature:
https://vueform.com/docs/handling-form-data#handling-success-responses

Though I like the idea of giving something that replaces the form when successfully submited, which could be a text in a default layout or something that could be replaced easily via a slot.

**Reply by letmaik** [reply]
I think something that replaces the form would be enough for most scenarios, typically a "Thank you, this is what happens next" kind of page. I would also be interested in figuring out how to do something completely custom after a form is submitted, meaning break out of vueform and switch to a different view/page in vue altogether. Maybe a little tutorial/demo on that as an "advanced" scenario would work, once the basic scenario is there.
